db-developments/bash/05_export.sh

```diff
@@ -1,11 +1,14 @@
 #!/bin/bash
 source bash/config.sh
+source sql/constants/export_schemas.sh
```

Comment:
Should this be stored somewhere else or named differently? This seemed like easiest way to not repeat a ton of code for the external tables by just storing the list of columns as a bash variable. See `_export.sql` to see why this was useful

Reply:
definitely nice to define things in one place!

I can't think of a better/simpler way to define and pass a column list to a bash-invoked sql file via `-v internal_columns="$internal_columns"`

but other potential file formats to store variables that come to mind are `.toml` and `.yaml`, but I don't know how straightforward it'd be to set variables in a bash script using those file formats

Reply:
future us: I think this is easier to do in either dbt + jinja or python land

Reply:
Agreed! Would like to get it out of bash eventually